airflow/utils/log/bluecore_bq_gcs_task_handler.py

```diff
@@ -0,0 +1,194 @@
+            # remote_log will contain error message.
+            remote_log = self.gcs_read(remote_loc, return_error=True)
+            log = '*** Reading remote log from {}.\n{}\n'.format(
+                remote_loc, remote_log)
+            return log
+
+        # everything above is copied from gcs_task_handler.py, the following elif statement includes changes
+        # specifically for bluecore_gcs_task_handler
+        elif ti.operator == "KubernetesJobOperator":
+            # If remote file is not available and the job is a Kubernetes Job,
+            # use the task instance attributes to query BigQuery logs and
+            # make the logs available in real time from the "Log" page of the UI.
+
+            # TODO:  job_name is always none on the first try, should fix this so an exception doesn't show each time
+            job_name = ti.xcom_pull(task_ids=ti.task_id, key='kubernetes_job_name')
+            if job_name is None:
+                raise Exception("There is no XCOM kubernetes_job_name. Something is exploding.")
+
+            pod_output = self.get_pods(job_name)
+
+            billing_project_id = configuration.get('core', 'kubernetes_bigquery_billing_project')
+            client = bigquery.Client(billing_project_id)
+
+            # get info for the BQ table name
+            data_project_name = configuration.get('core', 'kubernetes_bigquery_data_project')
+            # get BQ table's date (today).. based on insert time & not execution time.
+            bq_table_date = datetime.utcnow().date().strftime("%Y%m%d")
+            log_lines = []
+            tables = []
+            while ti.state == "running":
+                for pod in pod_output['items']:
+                    pod_id = pod['metadata']['name']
+                    for container in pod['spec']['containers']:
+                        container_name = string.replace(container['name'], '-', '_')
+                        # TODO: replace gke_logs with a variable
+                        bq_table_name = "{}.gke_logs.{}_{}".format(str(data_project_name), str(container_name),
+                                                                   str(bq_table_date))
+                        tables.append((str(bq_table_name), str(pod_id)))
+
+                query = ("\n UNION ALL \n ".join(
+                    [BQGCSTaskHandler.generate_query(bq_table_name=bq_table_name, pod_id=pod_id) for
+                     bq_table_name, pod_id in tables]) + "\n ORDER BY pod_id, logName, timestamp")
+
+                query_job = client.query(query)
+                result = query_job.result()
+
+                for row in result:
+                    log_lines.append(str("{}  {}".format(row.logName, row.textPayload)))
+
+                return "\n".join(log_lines)
+
+            # else statement from gcs_task_handler.py
+            else:
+                log = super(GCSTaskHandler, self)._read(ti, try_number)
```

Comment:
This should be log = super(BQGCSTaskHandler, self)._read(ti, try_number)

Reply:
Yes! changing it.